pageserver/src/tenant/timeline/span.rs

```diff
@@ -7,10 +7,8 @@ pub(crate) fn debug_assert_current_span_has_tenant_and_timeline_id() {}
 #[cfg(debug_assertions)]
 #[track_caller]
 pub(crate) fn debug_assert_current_span_has_tenant_and_timeline_id() {
```

Comment:
does the name now still reflect the purpose?

Reply:
Yes, the two extractors are still being used on lines L15R13, as in both `tenant_id` and `timeline_id` are being searched for; the commit removed searching for `tenant` and `timeline` keys respectively.